tools/astarte_e2e/lib/astarte_e2e/config.ex

```diff
@@ -50,130 +50,150 @@ defmodule AstarteE2E.Config do
   @type notifier_options :: [notifier_option()]
 
   @envdoc "Astarte Pairing URL (e.g. https://api.astarte.example.com/pairing)."
-  app_env :pairing_url, :astarte_e2e, :pairing_url,
+  app_env(:pairing_url, :astarte_e2e, :pairing_url,
```

Comment:
The previous formatting should be valid, as the `.formatter.exs` file includes the `:skogsra` entry. Could you check?